Project_2/LocationStatusAPI/src/main/java/com/Project_2_Location_Status_API/Services/CovidApiService.java

```diff
@@ -0,0 +1,38 @@
+package com.Project_2_Location_Status_API.Services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+
+@Service
+public class CovidApiService {
+
+    private static final String GET_COVID_API = "https://disease.sh/v3/covid-19/countries/";
+
+
+//    @Autowired
+//    private RestTemplate covidApiRestTemplate;
+
+    public ResponseEntity<String> getAllData(String country){
```

Comment:
I think we should change "getAllData" to something more descriptive like "getAllDataByCountry"